cloud: add a note in quick start (#9395)
pingcap/docs: tidb-cloud/tidb-cloud-quickstart.md

    The data import process will take 5 to 10 minutes. When the data import progress bar shows **Success**, you successfully import the sample data and the database schema in your database.

    > **Note:**
    >

>     > After clicking **Import**, a dialog box "Warning: Please Read Before Proceeding" is displayed, reminding that the import process will consume significant database resources. Make sure that you read the warning message before clicking **Confirm** on the dialog box.

Author: ```suggestion
    > After clicking **Import**, a dialog box "Warning: Please Read Before Proceeding" is displayed, reminding that the import process will consume significant database resources. You can ignore the warning message and go on clicking **Confirm** on the dialog box.
```
